apps/pollbook/backend/src/globals.ts

```diff
@@ -48,3 +48,18 @@ export const MAX_POLLBOOK_PACKAGE_SIZE = 10 * MEGABYTE;
 
 export const POLLBOOK_PACKAGE_ASSET_FILE_NAME = 'pollbook-package.zip';
 export const POLLBOOK_PACKAGE_FILENAME_PREFIX = 'pollbook-package';
+
+export const VOTER_INPUT_FIELD_LIMITS = {
+  firstName: 50,
+  lastName: 75,
+  middleName: 50,
+  nameSuffix: 5,
+  streetNumber: 6,
+  streetName: 50,
+  streetSuffix: 4,
+  apartmentUnitNumber: 15,
+  addressLine2: 75,
+  zip5: 5,
+  zip4: 4,
+  cityTown: 50,
+} as const;
```

Comment:
Just curious, how did we come up with these limits?

Reply:
I believe @adghayes set them based on the NH internal systems limits